Makefile

```diff
@@ -46,4 +48,12 @@ release:
 	opam publish submit $(NAME_VERSION)
 	rm -rf $(NAME_VERSION)
 
+install:
+	ocamlfind remove ppx_deriving
+	grep -E '^[[:space:]]+' ppx_deriving.install | \
```

Comment:
Any chance you could implement a tool using `opamlib`'s .install file parser? This was what I was planning some time ago, it would also get rid of cygwin dependency.